### DISCUSSION TITLE
决解了

### DISCUSSION THREAD

**Discussion by hacker873** [getlantern/lantern-forum-cn#292] [Q&A] [answered]
从下午开始就无法正常使用蓝灯了，程序无法启动，删除重装后变成了免费版本，在次授权到专业版，退出后还是无法再次启动，反复在重复这个循环。

**Comment by lexjhuang** [comment, marked as answer]
你好
今天我的付费版也是突然闪退一直打不开，经过一段调试后我写了一个bat脚本内容如下，之后我的付费版就可以正常启动了，希望这个有帮助！

del C:\Users\Administrator\AppData\Roaming\Lantern\global.yaml
Start C:\Users\Administrator\AppData\Roaming\Lantern\lantern.exe

**Reply by hacker873** [reply]
非常感谢分享，方法有效，已经解决了问题，我直接到蓝灯路径下删除了global.yaml文件，再重启程序授权下就正常使用了。我也尝试了几小时了，这文件也和免费版的文件对比过，没看出有不同呢。

**Comment by MaxMKMax** [comment]
@lexjhuang 
这位大哥, 谢谢您的帮忙.  各位用户也试试看这个方法. 

我们团队会尽快修复问题. 请稍等.

**Comment by PascalAndrews** [comment]
问题没有解决，删除yaml的配置之后就不能翻墙了……

**Reply by MaxMKMax** [reply]
@PascalAndrews 好消息，我们确实修复了 7.8.2 中的崩溃问题，版本 7.8.3 现已推出 https://github.com/getlantern/lantern
